CMakeLists_files.cmake

```diff
@@ -129,6 +134,10 @@ list (APPEND PUBLIC_HEADER_FILES
   opm/simulators/aquifers/AquiferFetkovich.hpp
   opm/simulators/aquifers/BlackoilAquiferModel.hpp
   opm/simulators/aquifers/BlackoilAquiferModel_impl.hpp
+  opm/simulators/linalg/BdaBridge.hpp
+  opm/simulators/linalg/BdaResult.hpp
```

Comment:
jenkins says that these headers are not there. Maybe they reside in `opm/simulators/linalg/bda`?

Reply:
Yes, I fixed the paths now.